students/WWhite/lesson06/test_mailroom4.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import unittest
+import mailroom4
+
+
+class TestMailroom(unittest.TestCase):
+
+    def setUp(self):
+        mailroom4.dict_donations = {
+            "bill gates": [25000.00, 3], "john snow": [10.56, 1], "monet holt": [50000.00, 2], "jeff bezos": [123500.09, 3],
+            "john wick": [120000.00, 2]
+        }
+
+    def tearDown(self):
+        pass
```

Comment:
You don't need to include tearDown in your test case if you're not adding any functionality